samples/openthread/common/overlay-ot-defaults.conf

```diff
@@ -20,6 +20,9 @@ CONFIG_NET_CONNECTION_MANAGER=y
 CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_INIT_STACKS=y
 
+# Shell stack
+CONFIG_SHELL_STACK_SIZE=3168
```

Comment:
does it compile with no Kconfig warning/error on NCP sample?

Reply:
Good point. In that case we need to introduce the change to zephyr. Working on that.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/34509